From 3287ae43935f5667eab5290830f8c3eb379d3cfe Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 27 Jan 2025 01:09:53 +0800
Subject: [PATCH 5/5] AOSCOS: mpv: drop unavailable entry to
 MpvProperties::AspectRatio

This was not yet implemented on the 0.12 branch.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 src/mpv/mpvpreview.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/mpv/mpvpreview.cpp b/src/mpv/mpvpreview.cpp
index b253e908..fc311bf7 100644
--- a/src/mpv/mpvpreview.cpp
+++ b/src/mpv/mpvpreview.cpp
@@ -13,7 +13,6 @@ MpvPreview::MpvPreview()
 {
     Q_EMIT setProperty(MpvProperties::self()->VO, QStringLiteral("libmpv"));
     Q_EMIT observeProperty(MpvProperties::self()->Position, MPV_FORMAT_DOUBLE);
-    Q_EMIT observeProperty(MpvProperties::self()->AspectRatio, MPV_FORMAT_DOUBLE);
 
     setProperty(MpvProperties::self()->Mute, true);
     setProperty(MpvProperties::self()->Pause, true);
-- 
2.48.1

